recipes/pydantic/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Please make sure to make a PR upstream to get them to include this in pypi.

Reply:
I manage the package. Why is this necessary? 

The licence is already state in setup.py file (twice), on the github page and in the docs. Isn't that enough?

Reply:
What I mean here is that the license file should be in the tarball from PyPi. This way conda-forge doesn't have to keep a copy here and can simply pull from pypi. Further, the pypi sdist or wheel needs to have a copy in order to be in compliance with the license, just like the conda package IIUIC.

To make sure it is included to pypi, all that needs to happen is to add an entry to the MANIFEST.in (a one line PR). 

We also don't need a new release of the package at all to merge. We can simply make the PR now, and then on the next release when the feedstock repo is updated, you can remove the license. Asking our contributors to do this is pretty standard practice. (I have done this more than once myself.)

Thank you so much for contributing your code to conda-forge! We really appreciate all of the work in building this ecosystem together!

Reply:
No problem.

I'm just wondering if it's really necessary, I'm also not sure whether you can still use `zip_safe=True` if you include such files, I'd also prefer to avoid adding yet another file to the root directory if possible, I'll experience with using `package_data` instead.

Reply:
Looks like it should be fine and `MANIFEST.in` is the only way to achieve what we want, more clutter but unavoidable.